src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.GlobalJsonReader.cs

```diff
@@ -61,7 +61,7 @@ static class GlobalJsonReader
                                     {
                                         case JsonTokenType.PropertyName:
                                             var sdkPropName = reader.GetString();
-                                            if (string.Equals("workloadVersion", sdkPropName, StringComparison.OrdinalIgnoreCase))
+                                            if (string.Equals("workloadSetVersion", sdkPropName, StringComparison.OrdinalIgnoreCase))
```

Comment:
My understanding was that we didn't have workload set version support for global.json yet, so I didn't think of it as a breaking change yet and preferred the more descriptive name. Both because it sounds like the current consensus is to use "workload version" instead of "workload set version" and because of what you said about updating VS, I don't think this is a good idea anymore, so I'll revert it.